lib/xgu/xgu.h

```diff
@@ -185,24 +192,38 @@ DWORD* xgu_enable_texture(DWORD* p, int stage_index, bool enabled) {
 
 inline
 DWORD* xgu_elements16(DWORD* p, uint16_t* elements, unsigned int count) {
-  assert(count % 2 == 0);
-  // This one is a bit complicated
-  assert(false);
-  return p;
+    assert(count % 2 == 0);
+    // This one is a bit complicated
+    assert(false);
+    return p;
 }
 
 inline
 DWORD* xgu_elements32(DWORD* p, uint32_t* elements, unsigned int count) {
-  // This one is a bit complicated
-  assert(false);
-  return p;
+    // This one is a bit complicated
+    assert(false);
+    return p;
 }
 
 #define xgu_draw_arrays(...) 0
 
 inline
 DWORD* xgu_set_alpha_test(DWORD* p, bool enabled) {
-  return push_command_boolean(p, NV097_SET_ALPHA_TEST_ENABLE, enabled);
+    return push_command_boolean(p, NV097_SET_ALPHA_TEST_ENABLE, enabled);
+}
+
+inline
+DWORD* xgu_set_transform_constant(DWORD* p, XGUVec4 *m, unsigned int count) {
```

Comment:
`m` sounds like matrix; use `v` [vector] or `c` [constant]

Reply:
Well, isn't the function supposed to be used with matrices sized x*4? 🤔

Reply:
Check what I explained on Discord earlier. This uploads an array of 4 component vectors [`XGUVec4`]. if you happen to set count to 4, then you have uploaded 4*vec4 = 4x4 matrix. if count is 1, then you might have uploaded for random scalar values packed into a float, or a color value, or whatever else.

*Strictly speaking, an array of vectors could always be interpreted as a "matrix", but I think this is confusing terminology in this case as it will also be used to upload independent vectors which are never accessed like a matrix.*

Reply:
You're right. Taken care of. :+1: